src/g_game.c

```diff
@@ -2963,9 +2981,11 @@ void G_ReloadDefaults(void)
   compatibility = false;     // killough 10/98: replaced by comp[] vector
   memcpy(comp, default_comp, sizeof comp);
 
-  demo_version = G_GetWadComplevel();
-
+  if (gamestate != GS_LEVEL)
```

Comment:
Not sure about this one. Is there a better/safer way to determine if the function is called from withing a running game (i.e. from the scope of a cheat code)?

Reply:
Perhapt check for `leveltime > 0` or `gameaction == ga_nothing` or just add a parameter to the function?

Reply:
I think function parameter is more readable/simple.

Reply:
This may interest you: https://github.com/kraflab/dsda-doom/commit/0092c134af94c6d26b4c4fa99e4d67600f5ddaaa

Reply:
Ah, you're looking in G_ReloadDefaults, not so easy. 😄 
Incidentally one of my next refactoring tasks is to fix what I think is chaos in how the source manages all these game configuration details, but that's for another day :^)